<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWidget</class>
 <widget class="QDialog" name="UserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>172</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add User</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Master Password:</string>
       </property>
       <property name="buddy">
        <cstring>txtPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtUserName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>User Name:</string>
       </property>
       <property name="buddy">
        <cstring>txtUserName</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Repeat Password:</string>
       </property>
       <property name="buddy">
        <cstring>txtPasswordRepeat</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtPasswordRepeat">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkCheckPassword">
     <property name="text">
      <string>Check Password when logging in</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtUserName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserWidget</receiver>
   <slot>checkInputValidity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserWidget</receiver>
   <slot>checkInputValidity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPasswordRepeat</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserWidget</receiver>
   <slot>checkInputValidity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkInputValidity()</slot>
 </slots>
</ui>
